man/plot_mod.Rd

```diff
@@ -3,13 +3,28 @@
 \encoding{UTF-8}
 \name{plot_mod}
 \alias{plot_mod}
+\alias{plot_mod.model_estimation}
+\alias{plot_mod.model_estimation_list}
+\alias{plot_mod.mixmod_regression}
+\alias{plot_mod.mixmod_regression_list}
+\alias{plot_mod.mixmod_em}
 \title{Add Estimated Population Line(s) to a Probability Plot}
 \usage{
-plot_mod(p_obj, x, ...)
+## S3 method for classes 'model_estimation', 'model_estimation_list',
+## 'mixmod_regression', 'mixmod_regression_list', 'mixmod_em'
+plot_mod(p_obj, x, title_trace = "Fit")
+}
+\arguments{
+\item{p_obj}{A plot object returned from \code{\link{plot_prob}}.}
```

Comment:
Maybe returned by

Reply:
At the moment "returned from" is the consistent expression. We'd have to change this at many locations, if necessary?